### PR TITLE
gameboy/mbc.cpp: Clarification about MBC3's "RTC"

### PR DESCRIPTION
**Pull request by wilbertpol**

### CODE REVIEW

**Comment by happppp** [comment]
Looks to me like dirtc can be used if you set month to 0?

File: src/devices/bus/gameboy/mbc.cpp [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
  Game Boy carts with MBC (Memory Bank Controller)
 
-
- TODO: RTC runs too fast while in-game, in MBC-3 games... find the problem!
+TODO:
+- The MBC3 RTC is not really a true RTC. The chip is used to track passed time even
+  when the console is off, not to keep track of the time of day. Keeping track of
+  passed time when the emulation was not running is currently not supported.
```
Comment (cuavas) [inline src/devices/bus/gameboy/mbc.cpp:10]: That’s exactly what an RTC does – it keeps track of time passing when the system is off.  This is exactly what `device_rtc_interface` is for.

Comment (happppp) [reply src/devices/bus/gameboy/mbc.cpp:10]: Nope, it insists to set day-of-month, and this thing has a 0-255 day counter instead. So you'd have to redo set_time at device_start.

**Comment by wilbertpol** [comment]
Going to redo this properly

**Pull request closed without merging**